src/app/components/publiclists/teamlist/teamcard/teamcard.component.ts

```diff
@@ -132,6 +170,15 @@ export class TeamcardComponent implements OnInit, OnChanges {
     this.deleteTeamCard.emit(this.team['id']);
   }
 
+    /**
+   * Fires delete member event.
+   */
+  deleteTeamMember(e, participantId) {
+    e.stopPropagation();
+    const teamid = this.team['id'];
```

Comment:
Can we remove this extra line and pass `this.team['id]` directly to the emit function as we are not using it anywhere else